src/modules/micrortps_bridge/micrortps_client/CMakeLists.txt

```diff
@@ -32,26 +32,22 @@
 ############################################################################
 
 if(NOT GENERATE_RTPS_BRIDGE MATCHES "off")
+
     find_program(FASTRTPSGEN fastrtpsgen
                  PATHS $ENV{FASTRTPSGEN_DIR})
     if(NOT FASTRTPSGEN)
-        message(WARNING "WARNING: Unable to find fastrtpsgen")
-        set(GENERATE_RTPS_BRIDGE off)
+        message(WARNING "WARNING: Unable to find fastrtpsgen. Building PX4 without RTPS bridge support")
```

Comment:
No because after there is a [return()](https://github.com/eProsima/Firmware/blob/2b71db3cdb1f01f702b747968746d0f851a5a1a4/src/modules/micrortps_bridge/micrortps_client/CMakeLists.txt#L48)